app/src/main/java/org/schabi/newpipe/settings/ContentSettingsFragment.java

```diff
@@ -109,19 +115,34 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
             return true;
         });
 
-        Preference setPreferredLanguage = findPreference(getString(R.string.content_language_key));
+        /*Preference setPreferredLanguage = findPreference(getString(R.string.content_language_key));
```

Comment:
Can you please add a comment that this is not needed due to the extractor changes and localization changes.